 syntax = "proto3";
 package tutorial;

 service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
    
    rpc LotsOfReplies (HelloRequest) returns (stream HelloReply) {}
 }

 message HelloRequest {
    string name = 1;
 }

 message HelloReply {
    string message = 1;
 }